sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -114,4 +147,19 @@ export default abstract class HttpBaseClient {
         return url;
     }
   }
+
+  /**
+   * Download file as blob using Axios http response.
+   * @param response
+   */
+  private downloadAsBlob(response: AxiosResponse<any>) {
```

Comment:
I was under the impression that we would not be duplicating this logic and that the hyperlink creation would happen inside the composable. Did something change from our last meeting or am I missing something?

Reply:
yup, I also had the same impression

Reply:
Do we still need this method here?